index.js

```diff
@@ -2,23 +2,34 @@
 'use strict';
 
 var autoprefixer = require('broccoli-autoprefixer');
-var defaults     = require('lodash').defaults;
+var defaults = require('lodash').defaults;
+var includes = require('lodash').includes;
 
 module.exports = {
-  name: 'ember-cli-autoprefixer',
-  included: function(app, parentAddon) {
-    this.app = app;
-    this.options = defaults(this.app.options.autoprefixer || {}, {
-      enabled: true
-    });
-    this.enabled = this.options.enabled;
-    delete this.options.enabled;
-  },
-  postprocessTree: function(type, tree) {
-    if ((type === 'all' || type === 'styles') && this.enabled) {
-      tree = autoprefixer(tree, this.options);
-    }
+    name: 'ember-cli-autoprefixer',
```

Comment:
Looks like you accidentally changed the indentation of the code in this file to be four spaces instead of two.